src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
+
+##### Message Bundle Keys and Selectors #####
+
+######Additions######
+* `textSizeDescr`
+* `textFontDescr`
+* `lineSpaceDescr`
+* `contrastDescr`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>speakChoiceLabel</code></td>
+    <td><code>speakDescr</code></td>
+</tr>
+<tr>
+    <td><code>tocChoiceLabel</code></td>
+    <td><code>tocDescr</code></td>
+</tr>
+<tr>
+    <td><code>simplifyChoiceLabel</code></td>
+    <td><code>simplifyDescr</code></td>
+</tr>
+</table>
+
+##### CSS Selectors #####
+
+######Additions######
+* `.flc-prefsEditor-contrast-descr`
+* `.flc-prefsEditor-linespace-descr`
+* `.flc-prefsEditor-textsize-descr`
+* `.flc-prefsEditor-textfont-descr`
```

Comment:
You should also have the selector name here. The messageBundle keys and selectors don't have to be the same, although they are in this case. Also, it's not clear what these map to. I'd do something like

* `textSizeDescr: ".flc-prefsEditor-textsize-descr"`

or 

```javascript
textSizeDescr: ".flc-prefsEditor-textsize-descr"
```

The same sort of thing should be in the changes section where you should show the selector name and the css selector used.